server/models/video-blacklist.js

```diff
@@ -0,0 +1,110 @@
+'use strict'
+
+const modelUtils = require('./utils')
+
+// ---------------------------------------------------------------------------
+
+module.exports = function (sequelize, DataTypes) {
+  const BlacklistedVideo = sequelize.define('BlacklistedVideo',
```

Comment:
You don't need `remoteId` and `localId` (so we don't define columns explicitly in this table). This table will just be composed by two columns: `id` and `videoId`. The first is implicit and the second will be declared in the `associate` method.